### ISSUE TITLE
Rum is looking for a new maintainer

### ISSUE THREAD

**Issue by tonsky** [tonsky/rum#197]
TL;DR Rum stays in support mode for now; looking for a more hands-on and proactive lead to keep up with React updates & API requests.

Last year I moved away from web development. On the web, time is counted in dog years, so that already feels like I am 7 years behind.

That lead to Rum stagnating a bit. It works just fine, mind you, there have been no major issues with it for a long time. Also, personally, I feel it is already in a good place feature-wise and capability-wise. If it was up to me, I would be happy just to leave it as it is.

But Rum is not just for me, there are many projects relying on it, and people working on them might feel different. I would love to see Rum evolving and keeping up with modern APIs, approaches, React versions and just best practices. Unfortunately, I personally have little time, opportunity and motivation to do it myself.
 
The solution I see is: I would love to transfer ownership to someone who actually uses Rum on a day-to-day basis and is willing to evolve it.

What I imagine is needed:

- Familiarity with Rum.
- Deep knowledge of React.
- Being invested in Rum (actually using it on a project).
- Willing to keep Rum up to date with React updates.
- Proactive in improving existing APIs and designing & implementing new ones. You will have to prioritize issues and define the scope of work for yourself.

First work items that come to mind are updating to 16.13.1 (Rum is currently on 16.2), then add #165 #167 #171 #176, suspence, hooks. I would be happy to consult on both design and implementation, of course.

If you want the job, please contact me here, on [twitter.com/nikitonsky](https://twitter.com/nikitonsky) or [niki@tonsky.me](mailto:niki@tonsky.me).

Before the transition is complete, Rum will stay on support mode. I will address any critical issues and push releases, but most likely will not add anything new.

**Comment by roman01la** [comment]
As discussed, I can take over the library. Even though I'm not using Rum daily anymore, based on my experience with building [UIx](https://github.com/roman01la/uix) I could work on the following:
- Improving Hiccup interpreter performance
- Migrating to newer React APIs (lifecycle deprecations and new public APIs)
- Evaluating and implementing modern React features (Hooks, Suspense, etc.)
- General maintenance

**Comment by tonsky** [comment]
Thanks @roman01la! That’s great news
